civictechprojects/static/css/_vars.scss

```diff
@@ -20,36 +20,104 @@ $color-disabled-border: #a8a8a8; // for e.g. a disabled state button border
 $color-grey-frame-border: #6b6e70; // from design guide
```

Comment:
is this one for any frame/container border?

Reply:
We can ignore it. This - and everything in this section - is an old color that needs to be phased out. My plan is to add most of the DS colors in this PR, all of them in a second PR, then remove all these old variable names.

Reply:
got it